app/src/main/res/layout/content_item.xml

```diff
@@ -7,30 +8,34 @@
     <variable
       name="htmlContent"
       type="CharSequence" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.itemviewmodel.ContentViewModel" />
   </data>
 
   <FrameLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="20dp"
-    android:layout_marginTop="@dimen/divider_margin_top"
-    android:layout_marginEnd="28dp"
-    android:layout_marginBottom="@dimen/divider_margin_bottom"
-    android:background="@drawable/content_blue_background">
+    android:layout_marginStart="24dp"
+    android:layout_marginEnd="24dp"
```

Comment:
Remove this as you have already introduced `app: app:layoutMarginEnd ` at the bottom

Reply:
removed